cdap-notifications/src/main/java/co/cask/cdap/notifications/feeds/NotificationFeedException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.notifications.feeds;
+
+/**
+ * Exceptions about notification feed.
+ */
+public class NotificationFeedException extends Exception {
```

Comment:
But both of them belongs to the notification system. Maybe you should have one called `NotificationException` as the parent and have sub types for feeds, publish, subscription, ... exceptions.